packages/ollama-utils/README.md

```diff
@@ -0,0 +1,56 @@
+# `@huggingface/ollama-utils`
+
+Various utilities for maintaining Ollama compatibility with models on Hugging Face hub.
+
+Documentation: https://huggingface.co/docs/hub/en/ollama
+
+## Chat template converter
+
+```ts
+import { convertJinjaToGoTemplate } from "@huggingface/ollama-utils";
+
+const MODEL_INFO_URL = "https://huggingface.co/api/models/bartowski/Llama-3.2-3B-Instruct-GGUF?expand[]=gguf";
+const modelInfo = await (await fetch(MODEL_INFO_URL)).json();
+console.log(modelInfo);
+/**
+ * {
+ *   gguf: {
+ *     chat_template: "here is the Jinja chat template",
+ *     bos_token: "...",
+ *     eos_token: "...",
+ *     [...]
+ *   }
+ * }
+ */
+const convertedTemplate = convertJinjaToGoTemplate(modelInfo.gguf);
+if (convertedTemplate) {
+  console.log(convertedTemplate.ollama);
+  /**
+   * {
+   *   template: "this is the converted template, compatible with Ollama",
+   *   tokens: [... list of special tokens],
+   *   params: {
+   *     stop: [... list of stop tokens or stop words]
+   *   }
+   * }
+   */
+} else {
+  console.error("Conversion is not successful");
+}
+```
+
+## How can I add a custom template?
+
+Most templates will be converted automatically. You can debug the output template using:
+- This space to retrieve the converted template: https://huggingface.co/spaces/ngxson/debug_ollama_manifest
```

Comment:
maybe we can do both in the same space?

Reply:
I couldn't figure out how to embed my script into gradio space, but will have a look later

(The template debugging space is static btw)